libqtile/confreader.py

```diff
@@ -75,6 +76,14 @@ def _init_deprecated(self, extensions=None, **settings):
                           "'extension_defaults'", DeprecationWarning)
             self.extension_defaults.update(extensions.get('dmenu', {}))
 
+    def _init_fake_screens(self, **settings):
+        " Initiaize fake_screens if they are set."
+        try:
+            value = settings['fake_screens']
+            setattr(self, 'fake_screens', value)
+        except KeyError:
+            pass
+
```

Comment:
Why not just adding `"fake_screens",` in the list of `settings_keys`?

Reply:
+1, I think that should work.